display_list/display_list_ops.h

```diff
@@ -246,6 +247,28 @@ struct SetImageColorSourceOp : DLOp {
   }
 };
 
+// 4 byte header + 80 bytes for the embedded DlImageColorSource
```

Comment:
Is that the right size?

Also, this should be added to the list of DlColorSource samples in display_list_unittests.cc (where you have to specify a buffer size for verifying that the pointer math and dl->bytes() come out right)

Reply:
I can't seem to find the test you're referring to. For now I deleted these lines. I'm a little concerned about how a `std::vector<std::shared_ptr>` will be measured across different platforms.

Reply:
std::vector should be 24 bytes on most modern platforms

Reply:
An example of the new color source type should be added to the unit tests at:

https://github.com/flutter/engine/blob/e8d5233a81b62833c5a9afe9b8cfe8ec35c7927f/display_list/display_list_test_utils.cc#L137